README.md

```diff
@@ -36,17 +36,20 @@ var importer = rouxSassImporter(sass.NULL, {
   pantrySearchPaths: [
     path.resolve('node_modules'),
     path.resolve('some/other/path')
-  ];
+  ]
```

Comment:
good call out -- this was just a mistake in the example code about how to use this module. 

I realized it when I went to check something and got a lint error when I grabbed this as a starting point, as it stands the example js won't run